docs/source/models/supported_models.rst

```diff
@@ -350,6 +350,11 @@ Text Embedding
     - :code:`BAAI/bge-multilingual-gemma2`, etc.
     - 
     - ✅︎
+  * - :code:`GritLM`
+    - GritLM
+    - :code:`parasail-ai/GritLM-7B-vllm`.
+    - 
+    -
```

Comment:
Since this model directly inherits from `LlamaForCausalLM`, LoRA and PP should be supported.

Reply:
Done. The pooler would not be distributed though in case of PP but I guess that's a small layer that is fine to skip.